bcl/src/bc_mpl3115a2.c

```diff
@@ -82,7 +82,7 @@ bool bc_mpl3115a2_get_pressure_pascal(bc_mpl3115a2_t *self, float *pascal)
 
     uint32_t out_p = (uint32_t) self->_reg_out_p_msb_pressure << 16 | (uint32_t) self->_reg_out_p_csb_pressure << 8 | (uint32_t) self->_reg_out_p_lsb_pressure;
 
-    *pascal = ((float) out_p) / 64.f;
+    *pascal = ((float) out_p) / 64.f; // 2**4
```

Comment:
Just this better reflects our coding style and comment is wrong - it is 2**5

`*pascal = (float) out_p / 64.f;`